packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -31,7 +33,7 @@ describe('test project paths', () => {
     it('should return a path to the state folder if the project has a root workspace', () => {
```

Comment:
I'm not sure that these "should be defined" tests are necessary (the one above this test).  I think there are compilation problems if the function is removed since it is referenced in the the rest of the functions below.  @gbockus-sf do these provide value since we get the same signal at compile time if the function is no longer defined?